.github/workflows/auto-update.yml

```
@@ -14,7 +14,7 @@ jobs:
       with:
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
     - name: update jpostcode data
-      run: bundle exec rake jpostcode:data:update_all
+      run: RUBYOPT=-w bundle exec rake jpostcode:data:update_all
```

https://docs.github.com/ja/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow
```suggestion
      run: bundle exec rake jpostcode:data:update_all
      env:
        RUBYOPT: "-w" 
```